### ISSUE TITLE
Laptop doesn't suspend on lid close

### ISSUE THREAD

**Issue by NomisIV** [hyprwm/Hyprland#626]
When closing the laptop lid in sway the my laptop would automatically suspend, but with Hyprland it just seems to turn off the screen. This would be acceptable if it wasn't for the fact that I specifically have configured logind to hybrid-sleep on lid close.

If I run `systemctl suspend` is suspends like normal, but ideally I would like to not have to run that command before closing the lid every time.

Is Hyprland doing something to inhibit the suspend command, and can that be disabled?

**Comment by vaxerski** [comment]
No. Hyprland does not interfere, or at least shan't as far as I know. It works on my laptop and many others' without any configuration.

Can you pass me a minimal log? Launch hl, close the lid, open the lid, exit hl.

**Comment by angeloscosta** [comment]
> Sorry for the noise. Hyperland is working correctly and the issue was something unrelated. I got it working now! :)

Hey, I have the same issue. What solved it in your case?

**Comment by NomisIV** [comment]
The issue, IIRC, was that I had forgot to configure physlock to activate on the hybrid-sleep.target (but it did activate on suspend.target, which was why it worked rhen I tried `systemctl suspend`), so whenever I opened the laptop I was presented with an unlocked computer. I misstook this for hyperland not working correctly, while it in fact was my lock screen that was the ultimate culprit.
